scripts/copy_from_upstream/patches/pqclean-kyber-armneon-yml.patch

```diff
@@ -16,7 +16,7 @@ index 28e2014..56a4b5f 100644
 +++ b/crypto_kem/kyber1024/META.yml
 @@ -37,7 +37,7 @@ implementations:
      - name: aarch64
-       version: https://github.com/neon-ntt/neon-ntt/tree/b011eeff3515fb168aa4dbaa671d760009d98dbb
+       version: https://github.com/neon-ntt/neon-ntt/tree/c6ffbcd1e5590e4dddbc604fc80d2ad756485b4d
        supported_platforms:
 -        - architecture: arm_8
 +        - architecture: ARM64_V8
```

Comment:
This "architecture" difference makes me uneasy. Can you explain this, @Martyrshot or @bhess ?

Reply:
I think it's just a matter of us using this string directly in places that expect ARM64_V8 whereas PQClean happens to populate it with arm_8.  @Martyrshot?

Reply:
I wonder if this part of `update_upstream_alg_docs.py` will still work as expected, as it seems to expect `arm_8` to take the branch:

https://github.com/open-quantum-safe/liboqs/blob/2c98b97fa468cba89e11cf469613ed255c86bce1/scripts/copy_from_upstream/update_upstream_alg_docs.py#L209

Reply:
> I think it's just a matter of us using this string directly in places that expect ARM64_V8 whereas PQClean happens to populate it with arm_8. @Martyrshot?

Yes, exactly. It's just so we can use that string directly in our build system. Otherwise we'd have to add a check and replace everywhere that string is used.

> I wonder if this part of update_upstream_alg_docs.py will still work as expected, as it seems to expect arm_8 to take the branch:

It should be ok, as that line is reading straight from the upstream meta which doesn't get updated until line 210.

Reply:
> It should be ok, as that line is reading straight from the upstream meta which doesn't get updated until line 210.

But the branch would also remove the "asimd" flag. Now that the upstream-yml is patched, shouldn't the patch remove this flag from the yml?

Reply:
> But the branch would also remove the "asimd" flag. Now that the upstream-yml is patched, shouldn't the patch remove this flag from the yml?

Oh good catch. It looks like the patch isn't being applied when the repo is first cloned. I have a fix the completely removes the yml patch. I can push it to this pr if you'd like @dstebila.

Reply:
Sure, go for it.

Reply:
This should be resolved now, and the yml patch has been removed.